knot.x/knotx/conf/openapi.yaml

```diff
@@ -42,3 +42,16 @@ paths:
           description: At least one procedure has reported a non-healthy stat
         '500':
           description: One procedure has thrown an error or has not reported a status in time
+  /assets/*:
```

Comment:
I think we should get rid of this endpoint and also remove any additional assets.
In production scenarios, Knot.x will be not serving static assets.